README.md

```diff
@@ -32,4 +32,6 @@ On `require`, it is indexed and loaded into memory. Finding postal codes is just
 ## Alternatives in other languages
 
 These community-maintained libraries are based on the same dataset and offer equivalent features in other languages:
-- [Dart](https://pub.dartlang.org/packages/code_postaux)
+
+- [Dart](https://pub.dartlang.org/packages/code_postaux) by [@Kleak](https://github.com/Kleak/code_postaux).
```

Comment:
the pub.dartlang.org is here to inform the person on everything there is a link to the repos if needed and a link to the documentation also :)